security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_check_for_rule_updates.php

```diff
@@ -198,26 +198,26 @@ function snort_download_file_url($url, $file_out) {
 		curl_setopt($ch, CURLOPT_HEADER, false);
 		curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
 		curl_setopt($ch, CURLOPT_SSL_CIPHER_LIST, "TLSv1.2, TLSv1");
-		curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, $config['installedpackages']['snortglobal']['curl_no_verify_ssl_peer'] == "on" ? false : true);
+		curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, config_get_path('installedpackages/snortglobal/curl_no_verify_ssl_peer') == "on" ? false : true);
 		curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 15);
 		curl_setopt($ch, CURLOPT_TIMEOUT, 0);
 
 		// Honor any system restrictions on sending USERAGENT info
-		if (!isset($config['system']['do_not_send_host_uuid'])) {
+		if (!config_path_enabled('system', 'do_not_send_host_uuid')) {
 			curl_setopt($ch, CURLOPT_USERAGENT, $g['product_name'] . '/' . $g['product_version'] . ' : ' . get_single_sysctl('kern.hostuuid'));
 		}
 		else {
 			curl_setopt($ch, CURLOPT_USERAGENT, $g['product_name'] . '/' . $g['product_version']);
 		}
 
 		// Use the system proxy server setttings if configured
-		if (!empty($config['system']['proxyurl'])) {
-			curl_setopt($ch, CURLOPT_PROXY, $config['system']['proxyurl']);
-			if (!empty($config['system']['proxyport']))
-				curl_setopt($ch, CURLOPT_PROXYPORT, $config['system']['proxyport']);
-			if (!empty($config['system']['proxyuser']) && !empty($config['system']['proxypass'])) {
+		if (!empty(config_get_path('system/proxyurl'))) {
+			curl_setopt($ch, CURLOPT_PROXY, config_get_path('system;proxyurl'));
```

Comment:
This looks like a typo - ';' instead of '/'

Reply:
Yep, typo. Fixed it.